<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormulaSettingPage</class>
 <widget class="QWidget" name="FormulaSettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>376</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="receipe_title_widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_title">
         <property name="text">
          <string>点位设置</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>302</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="receipe_context_widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_zhengcount">
           <property name="text">
            <string>正面点位数量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_zheng">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit {
	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_cecount">
           <property name="text">
            <string>侧面点位数量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ce">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit {
	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_backcount">
           <property name="text">
            <string>背面点位数量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_bei">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit {
	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>405</width>
     <height>252</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="receipe_title_widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_title_model">
         <property name="text">
          <string>模型设置</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>302</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="receipe_context_widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_bright">
           <property name="text">
            <string>高光面</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_bright_zheng">
           <property name="text">
            <string>正面模型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_bright_zheng">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit {
	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_bright_zheng">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_bright_ce">
           <property name="text">
            <string>侧面模型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_bright_ce">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit {
	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_bright_ce">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_dark">
           <property name="text">
            <string>哑光面</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_dark_zheng">
           <property name="text">
            <string>正面模型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dark_zheng">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit {
	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_dark_zheng">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>30</number>
         </property>
         <item>
          <widget class="QLabel" name="label_dark_ce">
           <property name="text">
            <string>侧面模型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dark_ce">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#lineEdit {
	border: 1px solid #eeeeee;
	border-radius: 4px;
	color: rgba(0,0,0,0.6);
	padding-left: 20px;
    font-size: 14px;
}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_dark_ce">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
